Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm

```diff
@@ -267,8 +267,15 @@ static void updateCustomAppearance(CALayer *layer, GraphicsLayer::CustomAppearan
     if (properties.changedProperties & LayerChange::FiltersChanged)
         PlatformCAFilters::setFiltersOnLayer(layer, properties.filters ? *properties.filters : FilterOperations());
 
-    if (properties.changedProperties & LayerChange::AnimationsChanged)
+    if (properties.changedProperties & LayerChange::AnimationsChanged) {
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+        if (layerTreeHost->threadedAnimationResolutionEnabled()) {
+            LOG_WITH_STREAM(Animations, stream << "RemoteLayerTreePropertyApplier::applyProperties() found " << properties.animationChanges.effects.size() << " effects on layer " << layerTreeNode->layerID());
```

Comment:
In logs I'd prefer to see the layerID first.

Reply:
Will bring the layer ID up in the logging message when landing.